unskript-ctl/unskript-client.py

```diff
@@ -1449,6 +1523,8 @@ def stop_debug():
                         help='Run the given runbook FILENAME [-RUNBOOK_PARM1 VALUE1] etc..')
     parser.add_argument('-rc', '--run-checks', type=str,
                         help='Run all available checks [all | connector | failed]')
+    parser.add_argument('-rs', '--run-suites', type=str,
```

Comment:
WE are not supporting this as of now, so suppress this.

Reply:
sure. Commenting the code out.